tests/annotation_layers/fixtures.py

```diff
@@ -0,0 +1,101 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
this looks great, more common way to make fixtures available is to use conftest:
https://www.tutorialspoint.com/pytest/pytest_conftest_py.htm

tests/annotation_layers/conftest.py - will make them automatically available to all files in the modules and submodules

Reply:
Thanks, I wasn't aware of this feature! I looked into the effort required for properly implementing this, and I believe it makes sense to do that in a separate follow-up PR, as I think I'll end up shuffling around quite a few files.